src/Filters.cpp

```diff
@@ -175,7 +175,7 @@ float Filters::update_voltage_supercap(float v) {
 }
 
 void Filters::correctAltitude(struct DataFrame &data){
-  if(data.NUM_SATS_GPS >= 6){
+  if( (data.NUM_SATS_GPS >= 6) && (data.GPS_LAST_NEW - data.TIME < 1000*60*5) && (fabs(data.ALTITUDE_GPS - data.ALTITUDE_BAROMETER) < 1000)) {
```

Comment:
Should be `data.TIME - data.GPS_LAST_NEW` I think

Reply:
:new_moon_with_face: